README.md

```diff
@@ -1,8 +1,5 @@
 # Flowcean
 
-[![Pipeline Status](https://collaborating.tuhh.de/w-6/forschung/agenc/agenc/badges/main/pipeline.svg)](https://collaborating.tuhh.de/w-6/forschung/agenc/agenc/pipelines/main/latest)
-![Coverage](https://collaborating.tuhh.de/w-6/forschung/agenc/agenc/badges/main/coverage.svg)
-
```

Comment:
Can we get pipeline status and coverage from the github actions?

Reply:
we might... there is very similar stuff like in gitlab. I suggest to open an issue for this and for now remove the broken badges from gitlab without replacement

Reply:
Fair enough. I opened issue #45 for this.